File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.java
class Solution {
    public int minOperations(int[] nums, int k) {
        int arr[]=new int[101];
        int c=0;
        for(int i:nums)
        {   
            if(i<k)
            return -1;
            else if(i>k && arr[i]==0)
          {  arr[i]=1;
           c++;
         }
        }


        return c;
    }
}

File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
     set<int> st;
     int i;
     for(auto &i:nums)
     {
        if(i<k)
        return -1;
        if(i>k)
        st.insert(i);
     }
     return st.size();
    }
};